src/components/Inputs/DateRangePicker/DateRangePicker.vue

```diff
@@ -606,6 +618,12 @@ export default {
     }
   }
 }
+
+.date-range-picker-disabled {
+  opacity: 0.5;
+  cursor: not-allowed !important;
+}
```

Comment:
nitpick (non-blocking): component-name-disabled is aways { opacity: 0.5; cursor: not-allowed; }... This could be a pb class... Just saying 😶